src/AppInstallerCLICore/PortableInstaller.cpp

```diff
@@ -121,7 +124,7 @@ namespace AppInstaller::CLI::Portable
             if (std::filesystem::remove(filePath))
             {
                 AICLI_LOG(CLI, Info, << "Removed existing file at " << filePath);
-                m_stream << Resource::String::OverwritingExistingFileAtMessage << ' ' << filePath << std::endl;
+                m_stream << Resource::String::OverwritingExistingFileAtMessage << ' ' << filePath.string() << std::endl;

```

Comment:
This actually fixes an issue in writing to the command line output gives `\\` instead of `\` and isn't an issue with the logging. The log files are correct, but the command line view isn't

Reply:
Oh, thanks for the explanation. I saw a couple of other places directly output the path as well, is it possible to add an overload like the logging one so other uses are also fixed and for future?

Reply:
Ah, no worry, it's generic stringstream, I guess we'll have to fix them one by one then. btw, please use u8string() to match what we log